### PR TITLE
fee recipient logic

### PR DESCRIPTION
**Pull request by vadiminc**

### CODE REVIEW

File: contracts/SSVNetwork.sol [hunk 1]
```diff
@@ -654,6 +654,10 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         return (feeChangeRequest.fee.expand(), feeChangeRequest.approvalBeginTime, feeChangeRequest.approvalEndTime);
     }
 
+    function feeRecipientAddress(address recipientAddress) external override {
```
Comment (andrew-blox) [inline contracts/SSVNetwork.sol:657]: in iSSV file you define this param as feeRecipientAddress but here as recipientAddress
Either name is fine but how does this actually work with mismatched names?

**Review by andrew-blox — CHANGES_REQUESTED** [review]

**Comment by andrew-blox** [comment]
Maybe add a test so we keep full coverage

File: contracts/SSVNetwork.sol (hunk 1 above)
Comment (vadiminc) [reply contracts/SSVNetwork.sol:657]: it doesn't matter.

**Comment by vadiminc** [comment]
> Maybe add a test so we keep full coverage

it has the test already

File: contracts/SSVNetwork.sol (hunk 1 above)
Comment (vadiminc) [reply contracts/SSVNetwork.sol:657]: oh, the file is new, forgot to add into the pr. check now

Comment (mtabasco) [inline contracts/SSVNetwork.sol:657]: not sure if this function should be called only from EOA or can be called from contracts too.
if its mandatory only for the former, then you can check:
```
if(msg.sender == tx.origin) {
    revert OnlyEOAAllowed();
}
```
cc @andrew-blox

Comment (andrew-blox) [reply contracts/SSVNetwork.sol:657]: I dont think its needed here but thanks @mtabasco for this article it helped explain the concept to me:
https://ethereum.stackexchange.com/questions/113962/what-does-msg-sender-tx-origin-actually-do-why

**Review by andrew-blox — APPROVED** [review]

**Review by mtabasco — APPROVED** [review]

File: contracts/ISSVNetwork.sol [hunk 2]
```diff
@@ -131,6 +131,11 @@ interface ISSVNetwork {
         Pod pod
     );
 
+    event FeeRecipientAddressAdded(
+        address ownerAddress,
```
Comment (lior-blox) [inline contracts/ISSVNetwork.sol:135]: @vadiminc @mtabasco Should this be indexed address?

Comment (mtabasco) [reply contracts/ISSVNetwork.sol:135]: It depends on the use case. If we want to allow search using `ownerAddress`, `recipientAddress` or both, yes, then should be indexed

**Pull request merged**